.editorconfig

```diff
@@ -0,0 +1,18 @@
+root = true
+
+[*]
+indent_style = space
+indent_size = 4
```

Comment:
Ahh the old tabs vs spaces, 2 vs 4 debate.  

As you can tell, I subscribe to the 4 spaces approach.  For three reasons:
* Its just old habit.  Its what I use at work everyday and it's what I'm used to.
* I would argue that 4 makes blocks easier to pick out at a glance and increases readability.
* And I choose spaces because it just looks better in atom/vim.

With that being said, it is a preference and this isn't my codebase so its not really up to me.  I will say that all the files I've converted, I have converted to 4 spaces.  It also appears that the original version were also in spaces.  So I just changed it to 4 for readability.  

If your curious, our company JS standards are in a repo. I more or less keep to these standards: https://github.com/thenerdery/javascript-standards.
We also have HTML/CSS standards here: https://github.com/thenerdery/html-css-standards